recipe/meta.yaml

```diff
@@ -5,11 +5,10 @@ package:
   version: {{ version }}
 
 source:
-  fn: openblas-{{ version }}.tar.gz
   url: https://github.com/xianyi/OpenBLAS/archive/v{{ version }}.tar.gz
   sha256: e64c8fe083832ffbc1459ab6c72f71d53afd3b36e8497c922a15a06b72e9002f
   patches:
-    - 0002-Fix-gfortran-detection-for-ctng-based-cross-compilers.patch    # [c_compiler == 'gcc']
```

Comment:
Can you revert this change?

Reply:
Sure. I was just testing locally and should've not pushed it.